producer-cpp/docker-rtsp/Dockerfile

```diff
@@ -32,6 +32,22 @@ WORKDIR /opt/amazon-kinesis-video-streams-producer-sdk-cpp/build/
 RUN cmake .. -DBUILD_GSTREAMER_PLUGIN=ON &&\
     make
 
+
+FROM ubuntu:18.04
+
+RUN apt update && \
+    apt install -y \
+    gstreamer1.0-plugins-base-apps \
+    gstreamer1.0-plugins-bad \
+    gstreamer1.0-plugins-good \
+    gstreamer1.0-plugins-ugly \
+    openjdk-8-jre &&\
```

Comment:
Nit: Space between && and \

Reply:
Also, curious. Is there a reason we are installing gstreamer again? Is it not possible to use `FROM builder` instead to avoid dual installations?

Reply:
Yes, it is possible to copy GStreamer related files from builder. I think some of these packages are not required for building SDK, I will check and optimize these duplications.